ocsp/ocsp_nonblock/README.md

```diff
@@ -0,0 +1,35 @@
+# OCSP Examples
+
+Online Certificate Status Protocol (OCSP) is used for obtaining the revocation status of an X.509 digital certificate.
+
+
+## OCSP non-blocking example
+
+This uses a google.com certificate chain to demonstrate validating revocation status using an OCSP server.
+
+The Google.com certificate defines OCSP in the X509v3 extension "Authority Information Access" section.
+
+Example usage:
+
+```sh
+$ ./configure --enable-ocsp CFLAGS="-DHAVE_IO_TIMEOUT -DWOLFSSL_NONBLOCK_OCSP"
+$ make
+$ sudo make install
+
+$ make
+# gcc -o ocsp_nonblock ocsp_nonblock.c -Wall -I/usr/local/include -Os -L/usr/local/lib -lwolfssl
```

Comment:
This is taken care of in the `Makefile`. No need to have it in the instructions.